pkg/constants/constants.go

```diff
@@ -29,13 +29,14 @@ const (
 	IGNITE_TIMEOUT = 10
 
 	// In-container path for the firecracker socket
-	SOCKET_PATH = "/tmp/firecracker.sock"
+	FIRECRACKER_SOCKET_PATH = "firecracker"
```

Comment:
Can we please avoid creating an extra directory for now to keep the code a bit simpler?
Please attach these sockets at the `/var/lib/firecracker/vm/$id/firecracker.sock` etc. path directly

Reply:
Sure thing